src/server/server.config.js

```diff
@@ -265,6 +265,12 @@ const conf = convict({
     format: '*',
     env: 'ROLLBAR_TOKEN',
     default: undefined
+  },
+  fuse: {
+    doc: 'Main url for fuse api',
+    format: '*',
+    env: 'FUSE_API',
+    default: 'https://explorer.fuse.io'
```

Comment:
dont put a default and make it a required, other wise we will miss it

Reply:
done